src/main.c

```diff
@@ -112,33 +117,44 @@ main (signed argc, char * argv []) {
 CURLcode
 paste (const struct ptpst_state * state) {
 
-    CURL * handle = curl_easy_init();
     CURLcode status = CURLE_OK;
+    CURL * handle = curl_easy_init();
 
     if ( !handle ) {
         fputs("Failed to get CURL handle", stderr);
         return CURLE_FAILED_INIT;
     }
 
+    struct curl_httppost * post = NULL;
+    struct curl_httppost * last = NULL;
+
```

Comment:
I wish I could keep these two down by the formadd, but then the compiler will complain due to the `goto cleanup`  before they’re declared.